docs/resources/samples/move-rows-across-tables.md

```diff
@@ -0,0 +1,247 @@
+---
+title: 'Move rows across tables using Office Scripts'
+description: 'Learn how to move rows across tables by saving filters, then processing and reapplying the filters.'
+ms.date: 03/30/2021
+localization_priority: Normal
+---
+
+# Move rows across tables by saving filters, then processing and reapplying the filters
+
+This script achieves the following:
+
+* Select rows from the source table where the value in a column is equal to _some value_.
+* Move all selected rows into another (target) table on another worksheet.
+* Reapply the relevant filters on the source table.
```

Comment:
I'm not sure the verb tense on these is right. Shouldn't it be "Selects", "Moves", and "Reapplies"?